.github/workflows/tests-playwright.yml

```diff
@@ -0,0 +1,35 @@
+name: Playwright tests
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      publish-folder:
+        type: string
+
+env:
+  PUBLISH_FOLDER: ${{ inputs.publish-folder || github.ref_name }}
+
+jobs:
+  run-playwright-tests:
+    name: Playwright Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        PROJECT: ["chromium", "firefox", "webkit"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Install Playwright
+        run: npx playwright install --with-deps
+
+      - name: Run Playwright tests
```

Comment:
If I'm not mistaken, the `playwright` workflow will use the docs built and published in the `pr` workflow (when run in a PR). However, when the docs are built in the `pr` workflow, it takes some minutes before the docs are actually available in the url. Did you have any problems with that here?

Reply:
Okay, great 🙏 I agree, let's see how this will behave over time.